bootstrap/web/iframe-adapter.js

```diff
@@ -53,75 +142,81 @@ function translateFunction(functionString, args){
     })
 }
 
-window.addEventListener("load", function () {
-    console.log('Loading iframe adapter instance');
-    this.window.top.postMessage('iframeload', '*');
-  }, false);
+function removeActionsFromContextMenu(itemsArray){
+    let copy = JSON.parse(JSON.stringify(itemsArray));
+    for(let i = 0; i < copy.length; i++){
+        if(copy[i].children){
+            copy[i].children = removeActionsFromContextMenu(copy[i].children)
```

Comment:
Just add a TODO here to give a little bit of info for whoever needs to deal with children properties for the ContextMenuItem[] structure so they'll know how to make it work/if they need to change things here to make it work